docs/platform/reference/tic-faq.md

```diff
@@ -0,0 +1,40 @@
+<!-- omit in toc -->
+# TIC Frequently Asked Questions
+
+- [What is the building block?](#what-is-the-building-block)
+- [What are the "Easy" and "Advanced" mode?](#what-are-the-easy-and-advanced-mode)
+- [How to get a recipe for TIC?](#how-to-get-a-recipe-for-tic)
+- [How to add my applications to the Tizen binary?](#how-to-add-my-applications-to-the-tizen-binary)
+- [How to add my platform packages to the Tizen binary?](#how-to-add-my-platform-packages-to-the-tizen-binary)
+- [Can I find artifacts of TIC in my host PC?](#can-i-find-artifacts-of-tic-in-my-host-pc)
+
+## What is the building block?
+
+- The building block is the package that defines a set of combined packages for a specific purpose.
+  You can see pre-defined building blocks and their structure in [the repository of building blocks](https://git.tizen.org/cgit/tools/building-blocks/tree/?h=tizen).
+  You can also define by yourself your own blocks by creating `.inc` files and adding them to the [building-block.spec](https://git.tizen.org/cgit/tools/building-blocks/tree/packaging/building-blocks.spec?h=tizen) file.
```

Comment:
Just noticed this, the "by yourself" part is redundant so suggesting you to removing it.
```suggestion
  You can also define your own blocks by creating `.inc` files and adding them to the [building-block.spec](https://git.tizen.org/cgit/tools/building-blocks/tree/packaging/building-blocks.spec?h=tizen) file.
```